### PR TITLE
made studant model

### PR DESCRIPTION
**Pull request by liammuir**
its kinda sus tho

### CODE REVIEW

File: StudentTracker/students/models.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
 from django.db import models
 
 # Create your models here.
+
+class Student(models.Model):
+    name = models.CharField(max_length=50)
+    email = models.EmailField(max_length=60)
+    document = models.CharField(max_length=50)
+    phone = models.CharField(max_length=15)
+    registrationDate = models.DateField()
```
Comment (davidtollefson) [inline StudentTracker/students/models.py:10]: It's not mentioned in the issue, but can you add auto_now_add=True to the params for DateField

**Review by davidtollefson — COMMENTED** [review]
Looks good, only one small change before approval

File: StudentTracker/students/models.py (hunk 1 above)
Comment (liammuir) [reply StudentTracker/students/models.py:10]: ok

**Review by davidtollefson — APPROVED** [review]

**Comment by davidtollefson** [comment]
Beauty, pls merge.

**Pull request merged**